CMakeLists.txt

```diff
@@ -29,11 +29,23 @@ target_compile_options(vk-bootstrap-compiler-warnings
         /W4>
         )
 
+target_include_directories(vk-bootstrap PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
+    $<INSTALL_INTERFACE:include>)
 target_include_directories(vk-bootstrap PUBLIC src)
 target_include_directories(vk-bootstrap PUBLIC ${Vulkan_INCLUDE_DIR})
 target_link_libraries(vk-bootstrap
         PRIVATE
-        vk-bootstrap-compiler-warnings)
+        vk-bootstrap-compiler-warnings
+        ${CMAKE_DL_LIBS})
```

Comment:
Ah thanks for catching this. 
For whatever reason while I knew DL_LIBS was needed, I didn't think to put it in the cmake...